test/WhereClauseTest.php

```diff
@@ -134,7 +135,7 @@ public function testSubstituteOnString(): void
     public function testSubstituteOnHash(): void
     {
         $a = new WhereClause(['name' => 'Tito', 'id'=> 1]);
-        $this->assertEquals("`name` = 'Tito' AND `id` = 1", $a->to_s($this->connection, substitute: true));
+        $this->assertEquals("{$this->name} = 'Tito' AND {$this->id} = 1", $a->to_s($this->connection, substitute: true));
```

Comment:
I don't understand while we still need this `$this->name` business if we've internalized the particulars of column name quoting to `PgsqlAdapter`.

Reply:
Because ```to_s()``` is the actual raw SQL, which is different for Postgres and MySQL.

Reply:
Then we're still failing at abstraction as far as this test is concerned. Can you find a different way to test the substitution, here?

Reply:
Alternatively, you could move this test to the adapter test cases (make sure you override as needed).

Reply:
Escaping of all columns is now deferred to SQLBBuilder::where()